catchup/service.go

```diff
@@ -70,10 +75,13 @@ type Service struct {
 
 	// The channel gets closed when the initial sync is complete. This allows for other services to avoid
 	// the overhead of starting prematurely (before this node is caught-up and can validate messages for example).
-	InitialSyncDone     chan struct{}
-	initialSyncNotified uint32
-	protocolErrorLogged bool
-	lastSupportedRound  basics.Round
+	InitialSyncDone              chan struct{}
+	initialSyncNotified          uint32
+	protocolErrorLogged          bool
+	lastSupportedRound           basics.Round
+	unmatchedPendingCertificates chan PendingUnmatchedCertificate
```

Comment:
The name unmatchedPendingCertificates is not very descriptive in the context of the catchup service. 
In the catchup context, this channel looks like a fast-track fetch block request channel. There is nothing specific to unmatched pending certificates in the catchup service. Right?

Reply:
Let me know if you have a better descriptive name for this channel and/or data structure.

Reply:
How about: immediateFetchCertificate?